### PR TITLE
Update CloudWatch section with high-resolution metrics and alarms

### PR DESCRIPTION
**Pull request by RyanGWU82**
High-resolution metrics and alarms were added in July. This PR updates the guide to include information about this option.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -870,7 +870,9 @@ CloudWatch
...
 * 🔸You can not use [more than one metric for an alarm](https://forums.aws.amazon.com/thread.jspa?threadID=94984).
 * 🔸Notifications you receive from alarms will not have any contextual detail; they have only the specifics of the threshold, alarm state, and timing.
-* 🔸Minimum granularity in CloudWatch is 1 minute. That means that multiple values of a metric that are pushed to CloudWatch within the same minute are aggregated into minimum, maximum, average and total (sum) per minute.
+* 🔸By default, CloudWatch metric resolution is 1 minute. If you send multiple vaules of a metric within the same minute, they will be aggregated into minimum, maximum, average and total (sum) per minute.
+* 🔸In July 2017, a new [high-resolution option](http://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/publishingMetrics.html#high-resolution-metrics) was added for CloudWatch metrics and alarms. This feature allows you to record metrics with 1-second resolution, and to evaluate CloudWatch alarms every 10 seconds.
```
Comment (jlevy) [inline README.md]: Add 🐥

File: README.md [hunk 2]
```diff
@@ -870,7 +870,9 @@ CloudWatch
 * 🔸Metrics in CloudWatch originate [on the hypervisor](https://forums.aws.amazon.com/message.jspa?messageID=403578). The hypervisor doesn't have access to OS information, so certain metrics (most notably memory utilization) are not available unless pushed to CloudWatch from inside the instance.
 * 🔸You can not use [more than one metric for an alarm](https://forums.aws.amazon.com/thread.jspa?threadID=94984).
 * 🔸Notifications you receive from alarms will not have any contextual detail; they have only the specifics of the threshold, alarm state, and timing.
-* 🔸Minimum granularity in CloudWatch is 1 minute. That means that multiple values of a metric that are pushed to CloudWatch within the same minute are aggregated into minimum, maximum, average and total (sum) per minute.
+* 🔸By default, CloudWatch metric resolution is 1 minute. If you send multiple vaules of a metric within the same minute, they will be aggregated into minimum, maximum, average and total (sum) per minute.
```
Comment (jlevy) [inline README.md]: Optional: Appropriate link here too?

Comment (RyanGWU82) [reply README.md]: Discussed IRL: That was in the original, and I'm not familiar with those rollups myself.

**Comment by jlevy** [comment]
LGTM!

**Pull request merged**